recipes/pytfa/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set name = "pytfa" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/EPFL-LCSB/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  #url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b4d848bd04cf8384b05e82fecb25a7cc59a65031ac5dee2c732853a12b086014
+
+build:
+  number: 0
```

This looks like it can be `noarch: python`:

```suggestion
   number: 0
  noarch: python
```

---

yes I think so